base/scripts/logging.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+echo "*.* @@192.168.66.2:24224" >> /etc/rsyslog.conf
+
```

Comment:
I would prefer if we simply skip logging configuration on the nodes completely in this repo. I would prefer if the consumer of our gocli infra and our clusters can tweak the log-collection in the dev-env or on CI to their needs with e.g. a daemonset: https://github.com/fluent/fluentd-kubernetes-daemonset/blob/master/fluentd-daemonset-syslog.yaml